docs/rules/accessibility/no-aria-label-misuse.md

```diff
@@ -2,32 +2,60 @@
 
 ## Rule Details
 
-This rule aims to minimize misuse of the `aria-label` and `aria-labelledby` attributes because the usage of these attributes is only guaranteed on interactive elements and a subset of ARIA roles. W3C provides [a list of ARIA roles which cannot be named](https://w3c.github.io/aria/#namefromprohibited) which is used as a basis for this linter.
+This rule aims to discourage common misuse of the `aria-label` and `aria-labelledby` attribute. `aria-label` and `aria-labelledby` support is only guaranteed on interactive elements like `button` or `a`, or on static elements like `div` and `span` with a permitted `role`. This rule will allow `aria-label` and `aria-labelledby` usage on `div` and `span` elements if it set to a role other than the ones listed in [WSC: a list of ARIA roles which cannot be named](https://w3c.github.io/aria/#namefromprohibited). This rule will never permit usage of `aria-label` and `aria-labelledby` on `h1`, `h2`, `h3`, `h4`, `h5`, `h6`, `strong`, `i`, `p`, `b`, or `code`.
 
-There are conflicting resources on what elements should support these naming attributes. For now, this rule will operate under a relatively simple heuristic aimed to minimize false positives, but has room for future improvements.
+### "Help! I'm trying to set a tooltip on a static element and this rule flagged it!"
 
-Learn more at [W3C Name Calcluation](https://w3c.github.io/aria/#namecalculation).
+Please do not use tooltips on static elements. It is a highly discouraged, inaccessible pattern for the following reasons:
```

Comment:
Do we want to include the source from where the reasons are coming?

Reply:
I added a source in line 20 :)